### PR TITLE
Onboarding modal

### PR DESCRIPTION
**Pull request by vikdoro**
Show the visitor an onboarding popup if they are on the /projects page and not logged in.
The modal should not be shown twice in 30 minutes.
https://trello.com/c/3fQpFX9O

### CODE REVIEW

**Comment by vikdoro** [comment]
[(3) Show overlay on the /projects page redirecting logged out users to /onboarding](https://trello.com/c/3fQpFX9O/1711-3-show-overlay-on-the-projects-page-redirecting-logged-out-users-to-onboarding)

File: src/elements/kw-app/kw-app.html [hunk 1]
```diff
@@ -334,12 +336,44 @@
...
+            _hasUserDoneOnboarding () {
+                return this.user || localStorage.getItem('onboarding');
+            },
+            _hasModalShownRecently () {
+                let onboardingToken = localStorage.getItem('onboarding-cta');
+                if (!onboardingToken) {
+                    return;
+                }
+                //true if shown in last 30 minutes
+                return (new Date().getTime() - parseInt(onboardingToken)) < 30000000
```
Comment (abridger) [inline src/elements/kw-app/kw-app.html]: Semi-colon at the end of the line

File: src/elements/kw-view-start/kw-view-start.html [hunk 2]
```diff
@@ -97,6 +98,7 @@
                 border: none;
                 border-radius: 30px;
                 padding: 15px 35px;
+                font-family: 'Bariol', sans-serif;
```
Comment (abridger) [inline src/elements/kw-view-start/kw-view-start.html]: Can we use the font declarations from `kano-styles` here?

File: src/elements/kw-onboarding-modal/kw-onboarding-modal.html [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+            :host kano-glint-animation a {
+                display: block;
+                position: relative;
+                background-color: transparent;
+                border: none;
+                border-radius: 3px;
+                font-size: 16px;
+                color: #fff;
+                background-color: #fe8233;
+                font-family: 'Bariol', sans-serif;
```
Comment (abridger) [inline src/elements/kw-onboarding-modal/kw-onboarding-modal.html]: Could we pull in the font-family from `kano-styles` here too?

**Comment by abridger** [comment]
Other than a few small things, and a tiny conflict, all looks good to me.

**Comment by vikdoro** [comment]
@abridger thank you!

**Pull request merged**